config/quota-management-list-configuration.yaml

```diff
@@ -91,6 +91,8 @@ registered_users_per_organisation:
         kafka_billing_models:
           - id: standard
             max_allowed_instances: 5
+          - id: marketplace
```

Comment:
Was that missing from another PR? it doesn't seem related to this PR

Reply:
I added that so that I could add tests to promote from `eval` to `marketplace`

Reply:
Is that used for something other than a test? It contains user `testuser1@example.com`, `testuser2@example.com`, etc so I thought it was used only for tests

Reply:
`config/quota-management-list-configuration.yaml` is used as the configuration of the quota management list.
It takes effect even on non-test runs.
I was assuming the existence of `testuser1...` was to give an example of the format, although that could be commented out. We should avoid setting the default configuration files with test data. Although this seems an issue that was already happening with this file as you mentioned. We can deal with it separately.